CodeGen/src/IrBuilder.cpp

```diff
@@ -381,6 +381,12 @@ void IrBuilder::translateInst(LuauOpcode op, const Instruction* pc, int i)
     case LOP_POWK:
         translateInstBinaryK(*this, pc, i, TM_POW);
         break;
+    case LOP_SUBRK:
+        translateInstBinaryRK(*this, pc, i, TM_SUB);
```

Comment:
Wouldn't KR make more sense?

Reply:
Or is is 'ReverseK' and not 'RegisterK' (side note: is it too late to ask what K even stands for?)

Reply:
ReverseKonstant